apps/synapse-portal-framework/src/components/elportal/ELContributeYourData.tsx

```diff
@@ -0,0 +1,97 @@
+import { Box, Link, Button } from '@mui/material'
+import React from 'react'
+import backgroundSpotsLeft from './assets/dot_blob_top_left.png'
+import backgroundSpotsRight from './assets/dot_blob_bottom_right.png'
+
+const ELContributeYourData: React.FC = () => {
+  return (
+    <Box
+      sx={{
+        backgroundColor: '#5BA998',
+        height: '250px',
+        textAlign: 'left',
+        display: 'flex',
+        alignItems: 'center',
+        position: 'relative',
+        overflow: 'hidden',
+      }}
+    >
+      <Box
+        sx={{
+          flex: '0 0 auto',
+          width: '33%',
+          height: '100%',
+          backgroundImage: `url(${backgroundSpotsLeft})`,
+          backgroundRepeat: 'no-repeat',
+          backgroundPosition: 'top left',
+          backgroundSize: '100%',
+          zIndex: 0,
+          overflow: 'clip',
+        }}
+      />
+      <Box
+        sx={{
+          flex: '1 1 auto',
+          zIndex: 1,
+          textAlign: 'center',
+          paddingTop: '100px',
+          paddingBottom: '100px',
+        }}
+      >
+        <Box
+          sx={{
+            width: '90%',
+            height: '3px',
+            backgroundColor: 'white',
+            opacity: 0.5,
+            marginBottom: 2,
+            margin: '0 auto',
+          }}
+        />
+        <h1 style={{ color: 'white', fontFamily: 'serif' }}>
+          Contribute Your Data
+        </h1>
+        <p style={{ color: 'white' }}>
+          If you are a funded portal contributor and ready to upload data to the
+          ELITE Portal, you can begin the data submission process by contacting
+          our data curation team through our service desk.
+        </p>
```

Comment:
Typography again, for consistency
```suggestion
        <Typography variant="body1" style={{ color: 'white' }}>
          If you are a funded portal contributor and ready to upload data to the
          ELITE Portal, you can begin the data submission process by contacting
          our data curation team through our service desk.
        </Typography>
```